lib/tasks/create_static_slots.rake

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+desc 'create some static slots'
+task create_some_static_slots: :environment do
+  common_days = %w[Wednesday Thursday Friday]
+  StaticSlot.week_types.each_key do |week_type|
+    StaticSlot.week_days.each_key do |week_day|
+      next unless common_days.include?(week_day)
+
+      create_static_slots_for_one_day(week_type, week_day)
+    end
+    create_static_slots_for_one_saturday(week_type)
+  end
+  puts 'creation done'
+end
+
+def create_static_slots_for_one_day(week_type, week_day)
+  create_static_slots_for_one_morning(week_type, week_day)
+  create_static_slots_for_one_afternoon(week_type, week_day)
+end
+
```

Comment:
Leave a comment to tell future readers that the date is actually not important here, and that this task is only a one-time use